src/main.js

```diff
@@ -691,25 +705,53 @@ class MiniGraphCard extends LitElement {
 
     this.updateQueue = [];
 
+    if (config.show.graph) {
+      this.entity.forEach((entity, i) => {
+        if (!entity || this.Graph[i].coords.length === 0) return;
+        this.Graph[i].y_axis = config.entities[i].y_axis === 'secondary' ? 1 : 0;
+      });
+    }
+
     this.bound = [
       config.lower_bound !== undefined
         ? config.lower_bound
-        : Math.min(...this.Graph.map(ele => ele.min)) || this.bound[0],
+        : Math.min(...this.Graph
+          .filter(ele => ele.y_axis === 0)
+          .map(ele => ele.min)) || this.bound[0],
       config.upper_bound !== undefined
         ? config.upper_bound
-        : Math.max(...this.Graph.map(ele => ele.max)) || this.bound[1],
+        : Math.max(...this.Graph
+          .filter(ele => ele.y_axis === 0)
+          .map(ele => ele.max)) || this.bound[1],
+    ];
+
+    this.boundSecondary = [
+      config.lower_bound_secondary !== undefined
+        ? config.lower_bound_secondary
+        : Math.min(...this.Graph
+          .filter(ele => ele.y_axis === 1)
+          .map(ele => ele.min)) || this.boundSecondary[0],
+      config.upper_boundSecondary !== undefined
+        ? config.upper_boundSecondary
```

Comment:
Rename to `upper_bound_secondary`

Reply:
Oops - I started the coding as camel-case before I spotted the snake case. I missed changing this one. Also implied from this is I haven't yet tested the upper/lower bounds on the secondary access, which I will also do.

Reply:
Done